.istanbul.yml

```diff
@@ -0,0 +1,6 @@
+instrumentation:
+    excludes:
+        - spec/**/*.js
+        - lib/index.js
+        - bin/manual-testing/*
```

Comment:
It appears that you have not updated this file to reflect the new location for the files.